88.md

```diff
@@ -0,0 +1,79 @@
+NIP-88
+======
+
+Recurring Subscriptions
+-----------------------
+
+`draft` `optional` `author:pablof7z`
+
+This NIP defines a way for a pubkey to create recurring subscription payments to another pubkey.
+
+## Tier Event
+A pubkey can create "tiers". These tiers might provide certain benefits to the supporters who subscribe to these.
+
+```json
+{
+    "kind": 7002,
+    "content": "<description of the tier>",
+    "tags": [
+        [ "title", "..." ],
+        [ "amount", "<amount-in-base-unit>", "currency", "<cadence>" ]
+    ]
+}
+```
+
+This event is generated by a pubkey who wants to allow users to support with different tiers.
+
+`.content` SHOULD be a description of what users who subscribe can expect.
+Tag `title` is an optional title for the tier.
+Tag `amount` MUST specify the payment required for this tier and its cadence.
+    * The first argument should be the stringified amount and the second argument the currency
+    * The third argument SHOULD be one of `daily`, `monthly`, `yearly`
+One or more `amount` tags MUST exist.
```

Comment:
How should changing the amount happen? A new `7002` ? Replace it? Reference the previous `7002` in the new `7002`? 

The question of reasoning here is to consider what might happen if denominated in sats and need to change that with huge price changes, or standard repricing of services. 

I wonder how to: 
1. historically track subscriptions previously no matter what the prices might have changed to
2. indiciate to the user that the previous pricing model has changed and they SHOULD NOT zap with the previous price without creating a new `7001` at least.

Reply:
I think it might just make sense to move this to NIP-33 events and the 7001 should `e` tag the version it's subscribed tro.